systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,357 @@
+        PodUtils.waitUntilMessageIsInPodLogs(deletePodName, "Successfully removed all " + topicsCountBelowQuota);
+
+        // List topics after deletion
+        resourceManager.createResource(extensionContext, adminClientListJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(listAdminName, NAMESPACE, 0);
+        listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        String afterDeletePodLogs = kubeClient().logs(listPodName);
+        assertThat(afterDeletePodLogs.contains(topicNamePrefix), is(false));
+        assertThat(afterDeletePodLogs, not(containsString(topicNamePrefix)));
+
+        JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        JobUtils.deleteJobWithWait(NAMESPACE, listAdminName);
+        JobUtils.deleteJobWithWait(NAMESPACE, deleteAdminName);
+    }
+
+    private String getAdminClientConfig(String kafkaUsername, int timeoutMs) {
+        final String saslJaasConfigEncrypted = kubeClient().getSecret(NAMESPACE, kafkaUsername).getData().get("sasl.jaas.config");
+        final String saslJaasConfigDecrypted = new String(Base64.getDecoder().decode(saslJaasConfigEncrypted), StandardCharsets.US_ASCII);
+        return "sasl.mechanism=SCRAM-SHA-512\n" +
+                "security.protocol=" + SecurityProtocol.SASL_PLAINTEXT + "\n" +
+                "sasl.jaas.config=" + saslJaasConfigDecrypted + "\n" +
+                "request.timeout.ms=" + timeoutMs;
+    }
+
+    void setupKafkaInNamespace(ExtensionContext extensionContext) {
+        // Deploy kafka with ScramSHA512
+        LOGGER.info("Deploying shared kafka across all test cases in {} namespace", NAMESPACE);
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(CLUSTER_NAME, 1)
+            .editSpec()
+                .editKafka()
+                    .withListeners(
+                        new GenericKafkaListenerBuilder()
+                            .withName(Constants.PLAIN_LISTENER_DEFAULT_NAME)
+                            .withPort(9092)
+                            .withType(KafkaListenerType.INTERNAL)
+                            .withTls(false)
+                            .withNewKafkaListenerAuthenticationScramSha512Auth()
+                            .endKafkaListenerAuthenticationScramSha512Auth()
+                            .build(),
+                        new GenericKafkaListenerBuilder()
+                            .withName(Constants.TLS_LISTENER_DEFAULT_NAME)
+                            .withPort(9093)
+                            .withType(KafkaListenerType.INTERNAL)
+                            .withTls(true)
+                            .withNewKafkaListenerAuthenticationTlsAuth()
+                        .endKafkaListenerAuthenticationTlsAuth()
+                        .build())
+                .endKafka()
+            .endSpec()
+            .build());
+        KafkaUtils.waitForKafkaReady(NAMESPACE, CLUSTER_NAME);
```

Comment:
This is not needed here, `resourceManager.createResource()` is waiting for readiness of all resources by default.

Reply:
Thanks, haven't noticed that.